code/game/machinery/computer/station_alert.dm

```diff
@@ -68,10 +71,13 @@
 				L -= I
 	return !cleared
 
-
 /obj/machinery/computer/station_alert/process()
+	air_alarm_count = LAZYLEN(alarms["Atmosphere"])

```

Comment:
I recall that my efforts to pull alarms using LAZYLEN outside of this file would give me undefined vars. I'm really wary of breaking this system again but I'll try it ONCE.